docs/using_deis/using-buildpacks.rst

```diff
@@ -27,6 +27,23 @@ Use ``deis create`` to create an application on the :ref:`controller`.
     Creating application... done, created unisex-huntress
     Git remote deis added
 
+Optional: Add SSH KEY to use private repositories
```

Comment:
Could we add a sentence or two here detailing *why* someone would need this? "To pull code from private repositories, you'll need to provide Deis with a private SSH key which has access. You can do so with deisctl:"